docker/README.md

```diff
@@ -65,7 +65,7 @@ $ fiftyone migrate --all
 $ fiftyone migrate --info
 FiftyOne Teams version: 0.14.4
 
-FiftyOne compatibility version: 0.23.0
+FiftyOne compatibility version: 0.23.1
```

Comment:
If we don't increment this now, I think that this block of text will not be changed during our sed on version bump?

```suggestion
FiftyOne compatibility version: 0.23.0
```

Reply:
Fixed